src/openeo_odc/map_processes_odc.py

```diff
@@ -104,6 +108,8 @@ def map_data(id, process, kwargs):
             params['reducer'] = {}
     else:
         params['data'] = convert_from_node_parameter(params['data'],
+    if 'target' in params: #target is used in resample_cube_spatial for example
```

Comment:
There is a syntax error on the above line ;)
line 110 there should be a `)` and no `,`